cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -363,12 +363,18 @@ chainSyncClient trce env queryVar metrics latestPoints currentTip actionQueue =
       ClientStNext
         { recvMsgRollForward = \blk tip ->
               logException trce "recvMsgRollForward: " $ do
-                Gauge.set (withOrigin 0 (fromIntegral . unBlockNo) (getTipBlockNo tip)) (mNodeHeight metrics)
-                details <- getSlotDetails trce env queryVar (getTipPoint tip) (cardanoBlockSlotNo blk)
-                newSize <- atomically $ do
-                            writeDbActionQueue actionQueue $ mkDbApply blk details
-                            lengthDbActionQueue actionQueue
-                Gauge.set (fromIntegral newSize) $ mQueuePostWrite metrics
+                if withOrigin 0 unBlockNo (getTipBlockNo tip) == 0
```

Comment:
The origin does not have block number 0, it has no block number.

The first two blocks have block number 0.

But why does this matter? It hardly matters what the tip is, what matters is what the block is, and that is not going to change by waiting.